backpack/__init__.py

```diff
@@ -195,7 +243,17 @@ def extend(module: torch.nn.Module, debug=False):
     module_was_already_extended = getattr(module, "_backpack_extend", False)
     if not module_was_already_extended:
         CTX.add_hook_handle(module.register_forward_hook(hook_store_io))
-        CTX.add_hook_handle(module.register_backward_hook(hook_run_extensions))
+        _register_hooks(module)
```

Comment:
[No action required for now] Discuss that there is no need to install hooks on container modules such as `Sequential`